css/responsive/desktop.styl

```diff
@@ -1,12 +1,15 @@
-$header-height = 90px
-$footer-height = 90px
+$header-height = 60px
+$footer-height = 48px
 
 // BASIC
 
 .main-content-padding
   padding-top: $header-height
   padding-bottom: $footer-height
 
+.min-height-content 
+  min-height: calc('100vh - ' + ($header-height * 2))
```

Comment:
I don't understand this line. Should it not be header+footer? This will make the min content height be 12px taller than the window no?